solr/core/src/java/org/apache/solr/cli/ConfigTool.java

```diff
@@ -92,12 +92,24 @@ public List<Option> getOptions() {
                 "Name of the Config API property to apply the action to, such as: 'updateHandler.autoSoftCommit.maxTime'.")
             .build(),
         Option.builder("v")
-            .longOpt("value")
+            .deprecated(
+                DeprecatedAttributes.builder()
+                    .setForRemoval(true)
+                    .setSince("9.8")
+                    .setDescription("Use --value instead")
+                    .get())
             .argName("VALUE")
             .hasArg()
             .required(false)
             .desc("Set the property to this value; accepts JSON objects and strings.")
             .build(),
+        Option.builder()
+            .longOpt("value")
+            .argName("VALUE")
+            .hasArg()
+            .required(false) // should be true when -v is removed.
+            .desc("Set the property to this value; accepts JSON objects and strings.")
+            .build(),
```

Comment:
`OptionGroup` could also be used here and the group may be marked as required, forcing the user to provide either option. This way, later checks (`if (value == null)`)are not necessary and `SolrCLI.getOptionWithDeprecatedAndDefault` may be replaced with `cli.getOption(optionGroup, null)`.